bot/cogs/filtering.py

```diff
@@ -478,7 +468,7 @@ async def _has_invites(self, text: str) -> Union[dict, bool]:
                 return True
 
             guild_id = guild.get("id")
```

Comment:
Would it make sense to use the invite converter here too? It's pretty similar code besides the `with_counts`.

Reply:
I experimented with this, but the problem with doing it is that the error messages raised by that converter make no sense in this context. For that reason I decided not to do it.

If you think I should still do it, I need to introduce some sort of complexity to handle that in a way that makes sense in a more generic context.

Reply:
That's fine. I've wanted to refactor the filtering cog eventually so I may take care of this when I get to that.